File: 2582/2582.c
#include <stdio.h>

int main() {
    // Dicionário que mapeia a soma dos botões para a música correspondente
    char *musicas[] = {
        "PROXYCITY",
        "P.Y.N.G.",
        "DNSUEY!",
        "SERVERS",
        "HOST!",
        "CRIPTONIZE",
        "OFFLINE DAY",
        "SALT",
        "ANSWER!",
        "RAR?",
        "WIFI ANTENNAS"
    };

    int C;
    scanf("%d", &C);

    for (int i = 0; i < C; i++) {
        int X, Y;
        scanf("%d %d", &X, &Y);
        int soma = X + Y;
        printf("%s\n", musicas[soma]);
    }

    return 0;
}

File: 2582/2582.php
<?php

// Dicionário que mapeia a soma dos botões para a música correspondente
$musicas = array(
    0 => "PROXYCITY",
    1 => "P.Y.N.G.",
    2 => "DNSUEY!",
    3 => "SERVERS",
    4 => "HOST!",
    5 => "CRIPTONIZE",
    6 => "OFFLINE DAY",
    7 => "SALT",
    8 => "ANSWER!",
    9 => "RAR?",
    10 => "WIFI ANTENNAS"
);

// Número de casos de teste
$C = intval(fgets(STDIN));

for ($i = 0; $i < $C; $i++) {
    // Ler os botões pressionados (X e Y)
    $input = explode(" ", trim(fgets(STDIN)));
    $X = intval($input[0]);
    $Y = intval($input[1]);

    // Calcular a soma dos botões e imprimir a música correspondente
    $soma = $X + $Y;
    echo $musicas[$soma] . "\n";
}

?>